prometheus/src/common.tsx

```diff
@@ -159,11 +212,11 @@ function CustomTooltipFormatBytes({ active, payload, label }) {
   return null;
 }
 
-function formatBytes(bytes) {
+function formatBytes(bytes: number) {
   if (bytes === 0) return '0 B';
   const units = ['B', 'KB', 'MB', 'GB', 'TB'];
-  const i = parseInt(Math.floor(Math.log(bytes) / Math.log(1024)));
-  return Math.round(bytes / Math.pow(1024, i), 2) + ' ' + units[i];
+  const i = Math.floor(Math.log(bytes) / Math.log(1024));
+  return Math.round((bytes / Math.pow(1024, i)) * 100) / 100 + ' ' + units[i];
```

Comment:
Is this * 100 / 100 correct? And do we need the space between number and unit? `100Mb vs 100 Mb`